### PR TITLE
Issue #521 Closes #521

### LINKED ISSUE #521: Multiple Guest Entries for 1 Person
### URL

https://www.jupiterbroadcasting.com/guests/page/2/
~https://www.jupiterbroadcasting.com/guests/page/4/~
https://www.jupiterbroadcasting.com/guests/page/6/

### On what type of device(s) did you see this bug?

Phone, Tablet, Desktop

### On which browser(s) are you seeing the problem?

Firefox, Chrome, Safari, Microsoft Edge

### Other Browser

_No response_

### Summary

I am noticing duplicate entries for guests on the show. These should probably be merged.

These are all the potentially duplicate guests I have seen:
- [cat](https://www.jupiterbroadcasting.com/guests/cat) and [catherine-kretzschmar](https://www.jupiterbroadcasting.com/guests/catherine-kretzschmar/)
- ~[heather](https://www.jupiterbroadcasting.com/guests/heather) and [heather-ellsworth](https://www.jupiterbroadcasting.com/guests/heather-ellsworth/)~
- [jeromelaban](https://www.jupiterbroadcasting.com/guests/jeromelaban) and [jlaban](https://www.jupiterbroadcasting.com/guests/jlaban)

### PR DESCRIPTION
**Pull request by 1701home**
Merched the following files and then updated the show notes for each episode they were in. #521 
Closes #521

### CODE REVIEW

**Review by CGBassPlayer — CHANGES_REQUESTED** [review]
Overall looks good. Just a couple of small things:

1. The profile image for Cat is much higher resolution and should probably be used in the merged profile.
2. The now unused images should be deleted from the images directory to help keep the repo size to a minimum (`static/images/people`)

**Comment by CGBassPlayer** [comment]
Don't forget about Jerome's duplicate image

**Comment by 1701home** [comment]
I will get that removed and resubmitted

**Review by CGBassPlayer — APPROVED** [review]
This LGTM.

Here are new merged people:
---
The new https://www.jupiterbroadcasting.com/guests/catherine-kretzschmar/

image
image
---

The new https://www.jupiterbroadcasting.com/guests/jeromelaban/
image
image

**Comment by 1701home** [comment]
Closes #512

File: docs/CONTRIBUTING.md
Comment (CGBassPlayer) [inline docs/CONTRIBUTING.md:1]: So it looks like this file is still here even after our chat the other day. I would try the git revert again, but I did forget 1 important thing about it. It needs to be `git revert <commit-hash>..HEAD` to get all the changes undone. [source](https://stackoverflow.com/a/8561889/13417053)

Shoot me a message if you need help with this.

**Review by CGBassPlayer — CHANGES_REQUESTED** [review]
Looks like the jlaban pictures were "undeleted" and the contributing is still here. If you need help getting this fixed up, feel free to reach out. Otherwise, just revert the contributing changes, then delete the images for jlaban

File: docs/CONTRIBUTING.md
Comment (1701home) [reply docs/CONTRIBUTING.md:1]: > So it looks like this file is still here even after our chat the other day. I would try the git revert again, but I did forget 1 important thing about it. It needs to be `git revert <commit-hash>..HEAD` to get all the changes undone. [source](https://stackoverflow.com/a/8561889/13417053)
> 
> Shoot me a message if you need help with this.

I understand. I will perform the outline steps and message you with any questions.

**Comment by 1701home** [comment]
I will make a new PR to close #521 via a new branch.

**Pull request closed without merging**